modules/go-health-monitoring.adoc

```diff
@@ -20,4 +20,6 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 . If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are:
 
 .. For `degraded`, the broken heart icon is displayed.
-.. For `out of sync`, the yellow yield icon is displayed.
\ No newline at end of file
+.. For `out of sync`, the yellow yield icon is displayed.
+
+. Click on the *Deployment History* tab to view the application deployment history. The page includes details such as, *Last deployment*, *Description* (commit message), *Environment*, *Outcome* (Succeeded/Failed), *Author* and *Revision*.
```

Comment:
To view the deployment history of an application, click on the *Deployment History* tab. The page includes details such as: *Last deployment*, *Description* (commit message), *Environment*, *Outcome* (Succeeded/Failed), *Author*, and *Revision*.

NOTE: If the texts within parenthesis () are part of UI, make them bold. If they are not, I suggest not using them at all.

Reply:
I think that is a fair comment - but no strong feelings either way.

Reply:
All UI elements should be in bold.